### PR TITLE
0.5.1 new

### PR DESCRIPTION
**Pull request by Rahul-Vijay**
- Hive
- Animations

### CODE REVIEW

**Comment by Rahul-Vijay** [comment]
@vikasprashar99 and @Rajat012k check this branch for any bugs or issues. Merge this PR, if there are no issues.

**Review by Rajat012k — APPROVED** [review]
Looks fine ! was issue with carasoul sorted!!(my contact has issues no product changed pls change my address)

**Comment by Rajat012k** [comment]
LGTM @Rahul-Vijay !!

File: lib/models/users.dart [hunk 1]
```diff
@@ -10,51 +10,64 @@
...
-    this.password,
-    this.mobile,
-    this.pincode,
-    this.address,
-    this.longitude,
-    this.latitude,
-  });
 
+@HiveType(typeId: 0)
+class UserModel extends HiveObject {
```
Comment (vikasprashar99) [inline lib/models/users.dart:21]: You have added hive for all responses?

**Comment by vikasprashar99** [comment]
Animation branch merged with this branch?

**Comment by vikasprashar99** [comment]
Everything working fine!

**Comment by vikasprashar99** [comment]
Everything working fine!

**Review by vikasprashar99 — APPROVED** [review]

**Pull request merged**